internal/pkg/bootstrap/handlers/secret/secret.go

```diff
@@ -103,3 +104,9 @@ func getSecretConfig(secretStoreInfo config.SecretStoreInfo) (vault.SecretConfig
 
 	return secretConfig, nil
 }
+
+// IsSecurityEnabled determines if security has been enabled.
+func (s *SecretProvider) IsSecurityEnabled() bool {
```

Comment:
Should this method be public?

Reply:
No, this does not need to be exported. Good catch.